reconcile/terraform_cloudflare_dns.py

```diff
@@ -213,10 +214,10 @@ def get_cloudflare_provider_rps(
         return DEFAULT_PROVIDER_RPS
     size = len(records)
     if size <= 50:
-        return 2
+        return min(-(-size // 50), DEFAULT_PROVIDER_RPS)
```

Comment:
This takes care of everything? if it's >= 200 it sets `DEFAULT_PROVIDER_RPS`, and `return LARGE_ZONE_RPS` is also then redundant? It replaces the need for if statements.

Reply:
`-(-size // 50)` is always `1` when `0 < size <= 50`, do you mean to use this formula under other conditions?

Reply:
This is really cool. I didn't realize the minus is executed before the floor division is.